cmd/vela-git/main.go

```diff
@@ -124,6 +124,11 @@ func main() {
 			Name:     "repo.remote",
 			Usage:    "the remote (clone URL) for the repo being cloned",
 		},
+		&cli.StringFlag{
+			EnvVars:  []string{"VELA_PULL_REQUEST_TARGET"},
```

Comment:
Not sure where the other environment variables come from referenced in other flags, so I just referenced the one from the docs [VELA_PULL_REQUEST_TARGET](https://go-vela.github.io/docs/reference/environment/variables/#pull_request-event-only). If there are others that need to be added please let me know :+1:

Reply:
We generally, add in those env vars like this:
https://github.com/go-vela/vela-git/blob/master/cmd/vela-git/main.go#L84

It allows for more customizability if someone wanted to use the plugin at a place that's not the beginning of the pipeline